workflow.md

```diff
@@ -0,0 +1,97 @@
+# Github and Team workflow for JuJa SQLcmd
```

Comment:
I don't see any reason that file to be in a project folder, it doesn't relate to a project. Recommend to exclude it from remote repository. To remove file from index, but keep it on HDD you should use git rm --cached